src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStat.module.scss

```diff
@@ -63,13 +64,27 @@
   @include govuk-media-query($from: tablet, $until: desktop) {
     flex: 0 1 50%;
     max-width: 50%;
+
+    // If there is only one key stat it should be full width
+    &:only-child {
+      flex: 1;
+      max-width: 100%;
+      text-align: center;
+    }
```

Comment:
Could put this after the media queries to avoid repeating this for each media query?

Reply:
Yep, cheers.